docs/src/main/asciidoc/_migration-guide-for-4.0.adoc

```diff
@@ -898,6 +899,35 @@ public class Demo {
 |com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders |com.azure.spring.servicebus.support.ServiceBusMessageHeaders
 |===
 
+==== From azure-spring-boot-starter-storage to spring-cloud-azure-starter-storage-blob
+
+.Class mapping from azure-spring-boot-starter-storage to spring-cloud-azure-starter-storage-blob
+[cols="<~,<~", options="header"]
+|===
+|Legacy class |Modern class
+|com.azure.spring.autoconfigure.storage.resource.AzureStorageProtocolResolver |com.azure.spring.core.resource.AzureStorageBlobProtocolResolver
+|com.azure.spring.autoconfigure.storage.resource.BlobStorageResource|com.azure.spring.core.resource.StorageBlobResource
+|com.azure.spring.autoconfigure.storage.resource.StorageRuntimeException|com.azure.spring.core.resource.StorageRuntimeException
+|com.azure.spring.autoconfigure.storage.resource.StorageType|com.azure.spring.core.resource.StorageType
+|com.azure.spring.autoconfigure.storage.StorageProperties|com.azure.spring.service.storage.common.StorageProperties
```

Comment:
Is this StorageProperties a public API to users? If not, should we skip this class here, to avoid confusing users?

Reply:
> Is this StorageProperties a public API to users? If not, should we skip this class here, to avoid confusing users?

these are public APIs, but I think we hope our users do not use these Apis directly.